src/content/docs/guides/Web development.mdx

```diff
@@ -416,6 +750,46 @@ Teraz jest bardzo jasne, że po kliknięciu na mięsień, zostaje też zupdate
 
 Ten przykład jeszcze nie jest taki zły, przy 1 useEffectcie dość łatwo się połapać co się dzieje, przy 11 robi się już kłopot i raz u mnie w pracy przez coś takiego padła produkcja 😭
 
+### Nieużywanie `useMemo`
+
+Ja wiem, że teraz będzie wchodził `React v19` ale zanim to, to minie jeszcze trochę czasu, plus aktualny compiler nie jest doskonały. Ale o co chodzi? Chodzi o to, że jeśli zrobisz coś takiego:
```

Comment:
aktualnie to za bardzo nie ma compilera do reacta 😭

Reply:
aktualny w sensie w react rc